examples/aws-cloudwatch/capacity_planning/ebs.juttle

```diff
@@ -0,0 +1,23 @@
+// This program can be used to guide decisions on capacity planning
+// for EBS volumes. It shows the average IO ops/second for all volumes
+// (combined) and compares that to the configured total Iops capacity
+// of the entire collection of volumes.
+//
+// If the total IO acticity is close to the Iops capacity, you may
+// want to consider adding more volumes or faster volumes to spread
+// the work across more items.
+
+(read cloudwatch -period 3600 -last :1h: product='EBS'
+ | filter name='VolumeReadOps' OR name='VolumeWriteOps'
```

Comment:
for cloudwatch I only support filtering by product or item and not other fields for now. I did update all the mysql reads to do the filtering in the read.